### PR TITLE
256 replace quiz cta with one for this summer

### PR DESCRIPTION
**Pull request by iamjamesgordon**
Quiz CTA replaced with Summer Activities imagery and text

- [Design ](https://xd.adobe.com/view/8ec8b611-be93-4fe4-905a-f5db948f868d-db9b/)
- [Trello Card](https://trello.com/c/6Kmn2Bu5/256-replace-quiz-cta-with-one-for-this-summer)

### CODE REVIEW

File: src/Components/QuizEncouragement.tsx [hunk 1]
```diff
@@ -61,23 +144,24 @@ const QuizButton = styled.div`
...
-          <QuizButton>Take our quiz</QuizButton>
+          <QuizButton>See what&apos;s on</QuizButton>
           <ImageContainer>
-            <Image
-              src="/img/try-quiz.png"
-              alt={''}
-              layout="fill"
-              objectFit="contain"
-            />
+            <img src="/img/summer_activities_mask.png" alt="" />
```
Comment (rosejbon) [inline src/Components/QuizEncouragement.tsx:162]: You could use the NextJS `Image` component, which is super cool and does things like lazy loading images when users internet is being slow - https://nextjs.org/docs/api-reference/next/image

File: src/Components/QuizEncouragement.tsx [hunk 2]
```diff
@@ -61,23 +144,24 @@ const QuizButton = styled.div`
...
   display: inline-block;
+
+  ${(props) => props.theme.screenSizes.tabletPortraitPlus} {
+    padding: 0.5rem 3.2rem;
+  }
 `
 
 export const QuizEncouragement = (): JSX.Element => {
   return (
     <Link href={`/quiz`} passHref>
```
Comment (rosejbon) [inline src/Components/QuizEncouragement.tsx:155]: Did you need to link the summer activities here?

**Review by rosejbon — APPROVED** [review]
Looks good to me James. Exciting to see all the changes happening, it is nice to see the app being used and updated. 🥳

**Review by Nikomus — APPROVED** [review]
Looks great! 🏖️

**Pull request merged**